cluster-autoscaler/metrics/metrics.go

```diff
@@ -433,6 +434,15 @@ var (
 			Help:      "Maximum evaluation time of a node being skipped during ScaleDown.",
 		},
 	)
+
+	scaleDownNodeRemovalLatency = k8smetrics.NewHistogramVec(
+		&k8smetrics.HistogramOpts{
+			Namespace: caNamespace,
+			Name:      "node_removal_latency_seconds",
+			Help:      "Latency from when an unneeded node is eligible for scale down until it is removed (deleted=true) or it became needed again (deleted=false).",
+			Buckets:   k8smetrics.ExponentialBuckets(1, 1.5, 18), // ~1s → ~1h
```

Comment:
nit: 18 buckets will result in last bucket boundary being ~985s, so ~16m, not 1h.